### DISCUSSION TITLE
minimal Mirador viewer only show image

### DISCUSSION THREAD

**Discussion by jpalacz** [ProjectMirador/mirador#3587] [Looking for examples]
Dear community,
I was wondering how to setup Mirador only to display one image, no UI for resources, no thumbnail navigation, no windows, or anything else. Basically the functionality a viewer like OpenSeadragon offers. Is that easily possible with Mirador and where to begin?

**Comment by marlo-longley** [comment]
Yes we want to make this a default setup. Thank you for recording the need.
